test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -673,6 +673,108 @@ context('action sidebar', function() {
       });
   });
 
+  specify('action comments', function() {
+    cy
+      .server()
+      .routePatient()
+      .routePatientActions()
+      .routePatientFlows()
+      .routeAction()
+      .routeActionActivity(fx => {
+        fx.data = [];
+        fx.data[0] = this.fxEvents[0];
+        fx.data[1] = this.fxEvents[1];
+
+        fx.data[0].attributes.date = now.format();
+        fx.data[1].attributes.date = moment(now).subtract(8, 'days').format();
+
+        return fx;
+      })
+      .routeActionComments(fx => {
+        fx.data = _.sample(fx.data, 3);
+
+        fx.data[0].relationships.clinician.data = { id: '11111' };
+        fx.data[0].attributes.edited_at = null;
+        fx.data[0].attributes.created_at = moment(now).subtract(2, 'days').format();
+        fx.data[0].attributes.message = 'Least Recent Message from Clinician McTester';
+
+        fx.data[1].relationships.clinician.data = { id: '11111' };
+        fx.data[1].attributes.edited_at = now.format();
+        fx.data[1].attributes.created_at = moment(now).subtract(1, 'days').format();
+        fx.data[1].attributes.message = 'Most Recent Message from Clinician McTester';
+
+        fx.data[2].relationships.clinician.data = { id: '22222' };
+        fx.data[2].attributes.created_at = moment(now).subtract(4, 'days').format();
+        fx.data[2].attributes.message = 'Message from Someone Else';
+        fx.data[2].attributes.edited_at = null;
+
+
+        return fx;
+      })
+      .routePrograms()
+      .routeAllProgramActions()
+      .routeAllProgramFlows()
+      .visit('/patient/1/action/12345')
+      .wait('@routePatient')
+      .wait('@routePatientActions')
+      .wait('@routeAction')
+      .wait('@routeActionActivity')
+      .wait('@routeActionComments');
+
+    cy
+      .get('[data-activity-region]')
+      .find('.qa-activity-item')
+      .eq(1)
+      .should('contain', 'CM')
+      .should('contain', 'Clinician McTester')
+      .should('contain', 'Edit')
+      .should('contain', 'Most Recent Message from Clinician McTester')
+      .should('contain', '(Edited)');
+
+    cy
+      .get('.js-edit')
```

Comment:
can this be a find within a get of something specific to this area.. if we add a js-edit to the main menu or something in the future I don't want the test to break.